terraspace/app/stacks/peer-kubernetes-components/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "iam_roles" {
+  value = {
+    for iam_role_arn, attributes in module.iam_assumable_role_admin :
+    iam_role_arn => attributes.iam_role_arn
+  }
```

Comment:
lets talk about what this `for` loop ends up generating.

Reply:
This is no longer necessary. Removed.
This was used by Terratest so it could assert if the roles were properly created. As we're not sure about how tests are going to be structured now (or if there are going to be tests at all) I won't even bother making this 'sensitve' and just removed to hide the arn's.